src/styles/blocks/catalog.scss

```diff
@@ -22,4 +23,11 @@
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    color: #34568b;
+    transition: var(--min-time);
```

Comment:
not fixed, specify what do you want to animate, `transition: transform $min-time`